docs/faq/env_var.md

```diff
@@ -309,6 +309,17 @@ If ctypes is used, it must be `mxnet._ctypes.ndarray.NDArrayBase`.
     with float32.
   - Model accuracies do not necessarily improve with this environment variable turned on.
 
+* MXNET_USE_FUSION
```

Comment:
As I suggested in dev@, could we align the variable to MXNET_SUBGRAPH_BACKEND to make the easy usage for the user? Currently, this env is for CPU operator fusion.

Reply:
I thought about it but I'm not sure - this fusion does not really use subgraph API because of its limitations (like Gluon support, separate forward and backward fusion), and would not work with the "get_optimized_symbol" API (which using this env variable would imply).

Reply:
It makes sense. 

@aaronmarkham  do you have any suggestion?
Simple background, @ptrendx please correct me if anything is not correct.

Currently, two environments deliver similar functionality where `MXNET_SUBGRAPH_BACKEND=MKLDNN` is for CPU fusion and `MXNET_USE_FUSION` is for GPU.
I suggest aligning to `MXNET_SUBGRAPH_BACKEN={MKLDNN, CUDNN}` because this is already widely used for operator fusion but the name of `SUBGRAPH` doesn't match with technical details inside GPU implementation.